web/src/index.css

```
@@ -994,3 +1043,179 @@ html.dark .with-pastel-balls::before {
+}
+
+.sidebar-nav-item:hover {
+  background-color: rgba(var(--semi-blue-0), 0.06);
+}
+
+.sidebar-nav-item-selected {
+  background-color: rgba(var(--semi-blue-0), 0.1);
+}
+
+/* ==================== 首页流动渐变背景 ==================== */
+@keyframes gradientFlow {
```

Stylelint (17.5.0)

[error] Expected keyframe name "gradientFlow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)